integration/spark-common-cluster-test/src/test/scala/org/apache/carbondata/cluster/sdv/generated/SetParameterTestCase.scala

```diff
@@ -252,6 +252,27 @@ class SetParameterTestCase extends QueryTest with BeforeAndAfterAll {
     sql("RESET")
   }
 
+  test("TC_014-test mv after reset properties") {
+    sql("drop table if exists maintable")
+    sql("drop MATERIALIZED VIEW if exists mv1")
+    sql("CREATE TABLE maintable(empno int,empname string,projectcode int, projectjoindate " +
+        "Timestamp, projectenddate date,salary double) STORED AS carbondata")
+    sql("CREATE MATERIALIZED VIEW mv1 as select timeseries(projectenddate,'day'), sum" +
+        "(projectcode) from maintable group by timeseries(projectenddate,'day')")
+    sql("insert into maintable select 1000,'PURUJIT',00012,'2015-07-26 12:07:28','2016-05-20'," +
+        "15000.00")
+    sql("insert into maintable select 1001,'PANKAJ',00010,'2015-07-26 17:32:20','2016-05-20'," +
+        "25000.00")
+    sql("set carbon.input.segments.defualt.maintable=1")
+    checkExistence(sql("EXPLAIN select timeseries(projectenddate,'day'), sum(projectcode) from " +
```

Comment:
Can use 'TestUtil.verifyMVHit' method to verify if query hits mv or not

Reply:
ok added check with `verifyMVHit`.